General Purpose OCR

Luyao-Zhang-1 opened the pull request.

Luyao-Zhang-1 committed:
Create general-purpose-ocr.md

VivekSinghRawat-1 reviewed (commented):
gpo document is updated.

Luyao-Zhang-1 committed:
Edit images

Luyao-Zhang-1 committed:
Finish the initial review

Luyao-Zhang-1 committed:
Create logo-powered-by-abbyy.png

Luyao-Zhang-1 committed:
Fix broken links

Luyao-Zhang-1 committed:
Fix links

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
Move 1.4 into Configuration

Luyao-Zhang-1 committed:
Fix typos

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
Add screenshots of microflows

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
add the oxford comma

Luyao-Zhang-1 committed:
Fix anchor

Luyao-Zhang-1 committed:
Fix anchor

mahendra-mahajan-mendix committed:
Update general-purpose-ocr.md

updated screenshots of domain model, activity configurations with added attribute of "enable confidence". Rewording and rephrasing of instructions to capture the process of GPO activity usage.

Luyao-Zhang-1 committed:
Merge pull request #4835 from mahendra-mahajan-mendix/patch-14

Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
upload images

Luyao-Zhang-1 committed:
Update general-purpose-ocr.md

Luyao-Zhang-1 committed:
Merge branch 'development' into lz-general-purpose-ocr-doc

Luyao-Zhang-1 commented:
The app service will not be released. Therefore, I close the PR.
